lib/intelligent_foods/api_client.rb

```diff
@@ -12,23 +12,20 @@ def initialize(id: nil, secret: nil, client: nil)
     end
 
     def authenticate!
-      uri = URI("#{IntelligentFoods.base_auth_url}/oauth2/token")
-      request = Net::HTTP::Post.new(uri)
-      request["content-type"] = "application/x-www-form-urlencoded"
-      body = { "grant_type" => "client_credentials" }
+      uri = URI("#{IntelligentFoods.base_url}/token")
```

Comment:
looks like there are 2 PRs in the mix here? one is about domains and one is about changes to the authentication?